package.json

```diff
@@ -2,7 +2,7 @@
     "name": "vscode-azureappservice",
     "displayName": "Azure App Service",
     "description": "An Azure App Service management extension for Visual Studio Code.",
-    "version": "0.17.1-alpha",
+    "version": "0.18.0",
```

Comment:
Hold off on bumping the version.  Since we'll be testing, we don't our telemetry to be populated with this version number until we actually release so we can track any errors that customers might have once we release

Reply:
You also want to hold off on the NOTICE.html because our dependencies change pretty often and it needs to reflect the dependencies we actually release with.

Honestly the "Prep for release" PR is pretty trivial and we usually do it a few days before release, not a week and a half before release